plaso/parsers/sqlite_plugins/mac_notificationcenter.py

```diff
@@ -0,0 +1,133 @@
+# -*- coding: utf-8 -*-
+"""This file contains a parser for the Notification Center database on MacOS.
+
+Notification Center events on MacOS are stored in a
+SQLite database file named "db", path is usually something like
+/private/var/folders/<W><d>/../0/com.apple.notificationcenter/db2/
+
+At the moment it takes into consideration only the main table, 'record'.
+Documentation of the behavior of each table still work in progress,
+current tables and supposed related content is the following:
+    Record: contains historical records
+    Requests: contain pending requests
+    Delivered: delivered requests
+    Displayed: displayed requests, by app_id
+    Snoozed: snoozed by user requests
+"""
+
+from __future__ import unicode_literals
+
+from dfdatetime import cocoa_time as dfdatetime_cocoa_time
+
+from plaso.containers import events
+from plaso.containers import time_events
+from plaso.lib import definitions
+from plaso.parsers import sqlite
+from plaso.parsers.sqlite_plugins import interface
+
+import biplist
+
+class MacNotificationCenterEventData(events.EventData):
+  """ MacOS NotificationCenter database event data
+
+  Attributes:
+    body (str): body of the notification message
+    bundle_name (str): name of the application's bundle that generated
+        the notification.
+    presented (int): either 1 or 0 if the notification has been shown to the
```

Comment:
Do you want to expand this in the formatter? See https://github.com/log2timeline/plaso/blob/master/plaso/formatters/twitter_android.py#L58 for an example

Reply:
good point. done